packages/react-composition/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@webiny/react-composition",
+  "version": "5.27.0",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/webiny/webiny-js.git"
+  },
+  "description": "A tiny composition framework for React components.",
+  "contributors": [
+    "Pavel Denisjuk <pavel@webiny.com>",
+    "Sven Al Hamad <sven@webiny.com>",
+    "Adrian Smijulj <adrian@webiny.com>"
+  ],
+  "license": "MIT",
+  "dependencies": {
+    "@babel/runtime": "^7.16.3",
+    "@types/react": "^16.14.0",
+    "lodash.debounce": "^4.0.8",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.16.0",
+    "@babel/core": "^7.16.0",
+    "@babel/preset-env": "^7.16.4",
+    "@babel/preset-typescript": "^7.16.0",
+    "@webiny/cli": "^5.27.0",
+    "@webiny/project-utils": "^5.27.0",
+    "jest": "^26.6.3",
+    "merge": "^1.2.1",
+    "rimraf": "^3.0.2",
```

Comment:
`merge` too?

Reply:
`jest` and `merge` are probably not necessary. `rimraf` I have to check, maybe we can remove it from all package deps (it's present everywhere), because it used to be in the package scripts, but now it's no longer a direct dep.